src/components/NavLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, ReactNode } from "react";
+import { Link } from "react-router-dom";
+
+interface Props {
+  to: string;
+  title: string;
+  children: ReactNode;
+}
+
+const NavLink: FC<Props> = ({ to, title, children }) => {
```

Comment:
Is the reason to have this component instead of `NavLink` from `react-router` so that we don't have to pass `className` prop?

Reply:
I added the component, because we have the custom check if the NavLink is active with 1 character less from the `to` prop. Otherwise the menu entry for "Instances" does not get active, when browsing from /instances to /instance/foo

Reply:
I see, got it :+1: